internal/page/donor/witnessing_your_signature.go

```diff
@@ -24,6 +24,11 @@ func WitnessingYourSignature(tmpl template.Template, witnessCodeSender WitnessCo
 			if lpa.Donor.CanSign.IsYes() {
 				return appData.Redirect(w, r, lpa, page.Paths.WitnessingAsCertificateProvider.Format(lpa.ID))
 			} else {
+				lpa, err := donorStore.Get(r.Context())
```

Comment:
why has this been added?

Reply:
Now we check the version of the Lpa on PUT we're getting a conditional expression error here as the PUTs happen so close together

Reply:
ah, I forgot that it does the `Put` in the code sender